.github/workflows/security.yml

```
@@ -0,0 +1,140 @@
+      with:
+        category: "/language:${{matrix.language}}"
+        upload: true
+
+  # Advanced Security Audit
+  security-audit:
+    name: Advanced Security Audit
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    
+    steps:
+    - name: Checkout repository
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)